tools/pr_checker/check_dns.py

```diff
@@ -0,0 +1,150 @@
+import click
+import dns.message
+import dns.name
+import dns.query
+import dns.resolver
+import re
+
+
+def read_rules(psl_filename):
+    """Read rules from a file into a set.
+
+    >>> sorted(read_rules('test_psl.dat'))
+    ['!main.bar.example.org', '*.bar.example.org', 'a.example.com', 'b.example.com.', 'example.com', 'example.org', 'foo.example.org']
```

Comment:
do we need a flat TLD and flat TLD w wildcard rule or not here?

ex: add ```'.com', '*.com'```

Reply:
Done!